### PR TITLE
Put TestEz under dev-dependencies in wally.toml

### LINKED ISSUE #62: TestEZ is listed as a regular dependency
TestEZ is listed as a regular dependency in the wally manifest, but should probably be a dev dependency.

### PR DESCRIPTION
**Pull request by Ukendio**
Resolves #62

### CODE REVIEW

**Comment by LastTalon** [comment]
This will need to update the project to place the dev packages somewhere during testing only as well as updating test runners to access it in the new location.

File: wally.toml [hunk 1]
```diff
@@ -5,5 +5,5 @@ registry = "https://github.com/UpliftGames/wally-index"
 realm = "shared"
 exclude = ["example", "docs", "Packages", ".moonwave", "build", "pages", "tests", "AttackOfTheKillerRoombas.rbxl", "AttackOfTheKillerRoombas.rbxl.lock", "matter.rbxl", "matter.rbxm", "roblox.toml"]
 
-[dependencies]
+[dev-dependencies]
 TestEZ = "roblox/testez@0.4.1"
```
Comment (LastTalon) [inline wally.toml:9]: There's no newline at the end here. Can you add one? That should be enforced by our .editorconfig now as well, can you check that that's working for you or if I might have messed up the file?

Comment (Ukendio) [reply wally.toml:9]: I have now pulled in the changes in main including the commits to .editorconfig, however that didn't seem to work properly? 

image

Instead I will just add the newline manually.

Comment (LastTalon) [reply wally.toml]: Do you have your extension configured correctly to use it? VS Code needs an extension since it doesn't support it natively.

**Review by LastTalon — APPROVED** [review]
This looks good to me. It should resolve things as they currently are. I still want to go through and give tests another pass later on though in order to get things better set up for automated testing.

File: wally.toml (hunk 1 above)
Comment (Ukendio) [reply wally.toml]: Oh wasn't aware of that, will check that out then

**Comment by Ukendio** [comment]
I think that squashed it 😬

**Pull request merged**